src/Umbraco.Infrastructure/Migrations/Install/DatabaseDataCreator.cs

```diff
@@ -1745,13 +1753,17 @@ private void CreatePropertyTypeData()
         }
     }
 
-    private void CreateLanguageData() =>
+    private void CreateLanguageData()
+    {
+        // Create the single language that's specified as the default UI language (by default, this is en-US).
+        var defaultCulture = new CultureInfo(_globalSettings.CurrentValue.DefaultUILanguage);
```

Comment:
As mentioned in https://github.com/umbraco/Umbraco-CMS/issues/12091 and fixed in https://github.com/umbraco/Umbraco-CMS/pull/12243, we should use `EnglishName` instead of `DisplayName`, so you get correct names that are also more predictable (doesn't depend on the current culture):

https://github.com/umbraco/Umbraco-CMS/blob/a5f363bbe283d06e629f24c40fb26fbe0121cab7/src/Umbraco.Core/Models/Language.cs#L36

Using `GetCultureInfo()` will also ensure it's a [cached, read-only instance](https://learn.microsoft.com/en-us/dotnet/api/system.globalization.cultureinfo.getcultureinfo?view=net-7.0#system-globalization-cultureinfo-getcultureinfo(system-string)):

```suggestion
        var defaultCulture = CultureInfo.GetCultureInfo(_globalSettings.CurrentValue.DefaultUILanguage);
```